projects/pulsar_adc/Readme.md

```diff
@@ -65,4 +65,11 @@ Here are some pointers to help you:
   * AD4010
   * AD4011
   * ADAQ4003
-  * AD4020
\ No newline at end of file
+  * AD4020
+
+build example:
+make FMC_N_PMOD=0 - builds PMOD version
+make FMC_N_PMOD=1 SPI_OP_MODE=0 - builds standard FMC version
+make FMC_N_PMOD=1 SPI_OP_MODE=1 - builds FMC version with SDO driven by SPI Engine CS and CS driven by GPIO
+make FMC_N_PMOD=1 SPI_OP_MODE=2 - builds FMC version with SDO driven by GPIO and CS driven by SPI Engine CS
```

Comment:
```suggestion
make FMC_N_PMOD=1 SPI_OP_MODE=1 - builds FMC version with ADC SDO pin driven by SPI Engine CS and ADC CS pin driven by GPIO
make FMC_N_PMOD=1 SPI_OP_MODE=2 - builds FMC version with ADC SDO pin driven by GPIO and ADC CS pin driven by SPI Engine CS
```

I got a little confused the first time I read this. Being more specific can help a little bit.